src/server/scripts/EasternKingdoms/Stratholme/stratholme.h

```diff
@@ -25,84 +25,89 @@
 
 enum DataTypes
 {
-    TYPE_BARON_RUN                      = 0,
```

Comment:
in enums it is customary to follow the convention that looks as follows:
```cpp
enum SomeEnum
{
    SOME_CONSTANT              = 0,
    SOME_WAY_LONGER_CONSTANT   = 1
};
```
So please revert these changes in formatting, as the original way easier to read